client/gutenberg/editor/controller.js

```diff
@@ -121,6 +130,11 @@ export const post = async ( context, next ) => {
 
 		const Editor = initGutenberg( userId, siteSlug );
 
+		// Reset the Gutenberg state
+		use( resetGutenbergState );
+		dispatch( 'core/edit-post' ).closePublishSidebar();
+		dispatch( 'core/edit-post' ).closeModal();
```

Comment:
What if we just added `core/edit-post` to the list of resettable stores?

Reply:
The `edit-post` store also contain other "harmless" parts and preferences that we are fine keeping across editor instances.